recipes/powershell/New-Policy-StepByStep.ps1

```diff
@@ -0,0 +1,426 @@
+<#
+.SYNOPSIS
+This sample script demonstrates the use of NetBackup Policy REST APIs.
+.DESCRIPTION
+This script can be run using NetBackup 8.1.2 and higher.
+It creates a policy with the default values for policy type specific attributes, adds a client, schedule, and backup selection to it, then deletes the client, schedule and finally deletes the policy.
+.EXAMPLE
+./New-Policy-StepByStep.ps1 -MasterServer "nb-master.example.com" -UserName "administrator" -Password "password" -DomainName "domain name"
+#>
+
+#Requires -Version 4.0
+
+Param (
+    [string]$MasterServer = $(Throw "Please specify the name of the NetBackup Master Server using the -MasterServer parameter."),

```

Comment:
can we rename  to nbmaster, username, password, domainName and domainType respectively to be consistent with other scripts?

Reply:
Tried to follow powershell script recommended way of defining command line arguments. I can change the MasterServer to NBMaster.